src/shared/api.ts

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Customization, Customizations } from '../codewhisperer/client/codewhispereruserclient'
+import {
+    getAvailableCustomizationsList,
+    getSelectedCustomization,
+    setSelectedCustomization,
+} from '../codewhisperer/util/customizationUtil'
+
+export interface AwsToolkitApi {
+    setCustomization(customization: Customization): Promise<boolean>
+    getSelectedCustomization(): Customization
```

Comment:
this is a toolkit general api so needs some "namespacing". can we export the codewhisperer-related API as a nested `codewhisperer` object ?


```suggestion
export interface CodeWhispererApi {
    setCustomization(customization: Customization): Promise<boolean>
    getSelectedCustomization(): Customization
    ...
}
export interface AwsToolkitApi {
    codewhisperer: CodeWhispererApi
}
```